docs/resources/core/security_list.md

```diff
@@ -58,10 +58,10 @@ resource "oci_core_security_list" "t" {
 The following arguments are supported:
 
 * `compartment_id` - (Required) The OCID of the compartment to contain the security list.
-* `display_name` - (Required) The OCID of the VCN.
+* `display_name` - (Optional) The OCID of the VCN.
```

Comment:
Fit and finish comment: We should update the description of the "display_name" argument to reflect the actual purpose of it.